org.mixedrealitytoolkit.input/Tests/Runtime/BasicInputTests.cs

```diff
@@ -41,9 +49,44 @@ public IEnumerator InputDeviceSmoketest()
             yield return null;
         }
 
+        // NOTE: Why is this test commented out? This test is for prefabs that have been migrated to use XRI 3 and since those were move to an Experimental folder then this test fails.
+        //       This test can be uncommented (probably update too) once the migrated XRI3 prefabs are used as the main prefabs.
+        ///// <summary>
+        ///// Ensures that the LeftHandController, RightHandController, and GazeController controllers have one and only one of the required components to work with com.unity.xr.interaction.toolkit 3.0.1 package
+        ///// </summary>
+        //[UnityTest]
+        //public IEnumerator XRI3ControllerHaveOneAndOnlyOneOfTheNeededXRI3ComponentsTest()
+        //{
+        //    var controllers = new[] {
+        //        CachedLookup.LeftHandController,
+        //        CachedLookup.RightHandController,
+        //        CachedLookup.GazeController
+        //    };
+
+        //    foreach (var controller in controllers)
+        //    {
+        //        Assert.That(controller, Is.Not.Null);
+
+        //        // Check all controllers have one and only one TrackedPoseDriver component
+        //        TrackedPoseDriver[] controllerTrackedPoseDrivers = controller.GetComponents<TrackedPoseDriver>();
+        //        Assert.AreEqual(controllerTrackedPoseDrivers.Length, 1);
+
+        //        // Check that LeftHandController and RightHandController also have one and only one XRRayInteractor component
+        //        if (controller.Equals(CachedLookup.LeftHandController) || controller.Equals(CachedLookup.RightHandController))
+        //        {
+        //            XRRayInteractor[] controllerXRRayInteractors = controller.GetComponents<XRRayInteractor>();
+        //            Assert.AreEqual(controllerXRRayInteractors.Length, 1);
+        //        }
+        //    }
+
+        //    yield return null;
```

Comment:
Even though these new XRI3 tests are commented out, it seems perhaps premature to be writing tests for the upcoming XRI3-based rig when we don't fully know what that rig will look like.  For instance, the LeftHandController, RightHandController, GazeController, which are deprecated XRBaseControllers, likely won't exist.  

Same for all new, commented out XRI3 tests below.

Reply:
To clarify the "existence" statement.  The controller class still exist, but will be deprecated.

I agree, rewriting this test will be required for a future PR.  Once the new rig is finished.

Reply:
Ok, removed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/716/commits/85c93e1e40f889128b6ef6324ac17bd23a68dd39 , resolving.